mobify/platform-scaffold
# WEB-1533: merlins alt tag audit

This PR adds missing alt tags to images on the site. It resolves images included directly in the project and images parsed from Merlins. The SFCC audit will be in a separate PR. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1533
 **Linked PRs**: n/a

## Changes
- Added alt tags to banners on the homepage
- Added alt tags to merlins PDP parsing
- Added alt tag to image in cart remove item modal
- Added alt tag to empty account order list image
- Added alt tag to image in account remove address modal
- Added alt tag to image in wishlist remove item modal

## Todos

## How to test-drive this PR
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- navigate through the site and check that no images are included without alt tags
